testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -184,11 +149,11 @@ def tearDown(self):
     def test_writer(self):
         "Test writing from a single frame PDB file to a PDB file." ""
         self.universe.atoms.write(self.outfile)
-        u = mda.Universe(PSF, self.outfile, permissive=True)
-        assert_almost_equal(u.atoms.positions,
-                            self.universe.atoms.positions, self.prec,
-                            err_msg="Writing PDB file with PrimitivePDBWriter "
-                            "does not reproduce original coordinates")
+        u = mda.Universe(PSF, self.outfile)
+        assert_almost_equal(u.atoms.coordinates(),
```

Comment:
Why has this gone back to coordinates()? Is it because we changed this to positions after you created this branch?